helm/cluster-azure/values.yaml

```diff
@@ -53,7 +56,7 @@ internal:
     name: giantswarm/kubectl
     registry: quay.io
     tag: 1.23.5
-  kubernetesVersion: 1.24.13
+  kubernetesVersion: 1.25.15
```

Comment:
We have `1.25.16` in the gallery but it has a `-gs` suffix. I didn't want to change so many things at once. We will need to update the image name template for newer versions. FYI.

Reply:
let's do it in two steps, first apply all changes with PSS/PSPs on 1.24, then ensure PSPs are gone (just in case), then 1.25

Reply:
deal